tests/scripts/github-action-helper.sh

```diff
@@ -79,13 +79,23 @@ function create_bluestore_partitions_and_pvcs_for_wal(){
 }
 
 function build_rook() {
+    build_type=build
+    if [ -n "$1" ]; then
+        build_type=$1
+    fi
     GOPATH=$(go env GOPATH) make clean
     # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
-    make -j$nproc IMAGES='ceph' VERSION=0 build
+    make -j$nproc IMAGES='ceph' VERSION=0 "$build_type"
     # validate build
     tests/scripts/validate_modified_files.sh build
     docker images
-    docker tag $(docker images | awk '/build-/ {print $1}') rook/ceph:master
+    if [[ "$build_type" == "build" ]]; then
+        docker tag $(docker images | awk '/build-/ {print $1}') rook/ceph:master
```

Comment:
We don't need to tag with build.all?

Reply:
We just don't know which one to tag since we get 2 images.

Reply:
I wonder if we don't even need to add the master tag, IIRC that was just needed for jenkins to use that tag for the integration tests. Do we still need it for this action? Or maybe we could try not adding the tag.

Reply:
The tag is there so that the operator deployment fetches the local image and not the one on the docker hub, so even with GitHub Actions we need to do that.

Reply:
Ok got it, this build is being called by the actions that also run the integration tests.